addon/services/lazy-loader.js

```diff
@@ -1,13 +1,20 @@
-/* globals require*/
 import Ember from 'ember';
 import bundles from 'ember-cli-bundle-loader/config/bundles';
-import config from 'ember-get-config';
+import { getContainer } from 'ember-cli-bundle-loader/utils/get-owner';
 
 const A = Ember.A;
-const loadedBundles = {};
-bundles.forEach(bundle=>loadedBundles[bundle.name] = false);
+let loadedBundles = {};
 
 export default Ember.Service.extend({
+  init () {
+    this._super(...arguments);
+    this.setBundles(bundles);
+  },
+  setBundles (bundles) {
+    loadedBundles = {};
+    bundles.forEach(bundle=>loadedBundles[bundle.name] = false);
+    this.set('bundles', bundles);
```

Comment:
if you're setting a value at a key on this object, should probably have `bundles: null` higher up to signal that that property exists ..?